config/urls.py

```diff
@@ -1,4 +1,5 @@
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
-]
+        path('user/', include('user.urls'))
```

Comment:
```suggestion
        path('user', include('user.urls'))
```
main url에서는 슬래시를 붙이지 않는 것이 wecode conding convention입니다 👍 
슬래시는 app url에서 공통적으로 다뤄주시는 것이 좋습니다 :)